README.md

```diff
@@ -57,6 +63,8 @@ def duplicates?(array)
 end
 ```
 
+ANSWER: O(n**2) - there is a loop within a loop, and each has a big-o of n
```

Comment:
Just FYI - I wouldn't use ** to represent power, as that's a ruby thing. Also syntax for Big O is O(n^2).